public/validators/main-albatross/NQ83 4MVH 53Q4 AL3B Q097 55GJ LUQ3 GSF0 85B7.json

```diff
@@ -0,0 +1,15 @@
+  "accentColor": "#754b47",
+  "fee": 0,
```

Comment:
Not the same as shown in the web page, either edit this or the web page info

Reply:
The web page info got updated.